modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,33 @@
+"""
+Modular Exponential
+"""
+
+def mod_exponent(base, power, mod):
+    """
```

Comment:
If i got it right them there is a need to change doc string in many python codes. :-)